### DISCUSSION TITLE
[dubbo-spi-extensions] Extension library compatible with Dubbo 3.1. x version

### DISCUSSION THREAD

**Discussion by songxiaosheng** [apache/dubbo#13832] [General]
At present, the extension library is already in the [3.2.0](https://github.com/apache/dubbo-spi-extensions/tree/3.2.0) branch  Dubbo 3.2. x version is compatible with the extension library, but it still needs to be compatible with other versions of the previous version, such as Dubbo 3.1. x version

**Comment by AlbumenJ** [comment]
Track in https://github.com/apache/dubbo/issues/13866
